code/modules/mob/mob_movement.dm

```diff
@@ -297,7 +297,7 @@
 ///For moving in space
 ///return TRUE for movement 0 for none
 /mob/Process_Spacemove(movement_dir = 0)
-	if(..())
+	if(spacewalk || ..())

```

Comment:
this will make the parent call never happen if spacewalk is true. intended?

Reply:
It was something I suggested in discord because the parent just checks the other ways something could be allowed to space walk. If a mob has spacewalk this will return true either way.

No actual work is done in this proc.

Reply:
:+1: